src/trip/components/tripsTable.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { getCredentials, register, unregister } from '../../login/store'
```

Comment:
So you are not unregistering ? It could be troublesome if the component is mounted/unmouted several times

Reply:
I guess a return in the first useEffect could help

Reply:
I should have put this PR as draft, actually it wasn't finalized, this part is now functionnal in the incoming push.